frontend/public/components/overview/pods-overview.tsx

```diff
@@ -156,41 +156,41 @@ export const PodsOverviewContent: React.SFC<PodsOverviewContentProps> = ({
     filteredPods = filteredPods.filter((pod) => !isPodWithoutImageId(pod));
   }
   filteredPods.sort(podCompare);
-
   const errorPodCount = _.size(_.filter(pods, (pod) => isPodError(pod)));
   const podsShown = Math.max(Math.min(errorPodCount, MAX_ERROR_PODS), MAX_PODS);
   const linkTo = allPodsLink || `${resourcePath(referenceFor(obj), name, namespace)}/pods`;
   const emptyMessage = emptyText || t('public~No Pods found for this resource.');
 
-  const podAlert = showWaitingForBuildAlert ? (
-    <Alert
-      isInline
-      variant="info"
-      title={t('public~Waiting for the build')}
-      actionLinks={
-        <AlertActionLink onClick={() => setShowWaitingPods(!showWaitingPods)}>
-          {showWaitingPods
-            ? t('public~Hide waiting pods with errors')
-            : t('public~Show waiting pods with errors')}
-        </AlertActionLink>
-      }
-    >
-      {t(
-        'public~Waiting for the first build to run successfully. You may temporarily see "ImagePullBackOff" and "ErrImagePull" errors while waiting.',
-      )}
-    </Alert>
-  ) : null;
-
   return (
     <>
       <SidebarSectionHeading text={t('public~Pods')}>
         {_.size(pods) > podsShown && (
           <Link className="sidebar__section-view-all" to={linkTo}>
-            {t('public~View all {{podSize}}', { podSize: _.size(pods) })}
+            {t('public~View all ({{podCount}})', { podCount: _.size(pods) })}
           </Link>
         )}
       </SidebarSectionHeading>
-      {buildConfigData?.loaded && !buildConfigData?.loadError && podAlert}
+      {showWaitingForBuildAlert ? (
+        <Alert
+          isInline
+          variant="info"
+          title={t('public~Waiting for the Build')}
+          actionLinks={
+            <AlertActionLink onClick={() => setShowWaitingPods(!showWaitingPods)}>
+              {`${
```

Comment:
This change looks unnecessary

Reply:
Yes.
Finished syncing the files from master branch. Thanks.